h/static/scripts/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "lib": ["es2018", "dom"],
+    "jsx": "react-jsx",
+    "jsxImportSource": "preact",
+    "module": "commonjs",
+    "noEmit": true,
+    "strict": true,
+    "target": "ES2020",
+    "useUnknownInCatchVariables": false
+  },
+  "include": ["**/*.js", "**/*.ts", "**/*.tsx"],
```

```suggestion
  "include": ["**/*.ts", "**/*.tsx"],
```

Limit type checking to TS / TSX files for now. That will make it easier to turn on typechecking in CI. You can also remove the `exclude` list as well.